Docs updates (#3079)
GoogleContainerTools/skaffold: docs/content/en/docs/pipeline-stages/testers.md

weight: 20
---

> It's common practice to want to validate built container images before deploying them to our cluster.

Reviewer: ```suggestion
It's common practice to validate built container images before deploying them to a cluster.
```
